dev/TabView/TabViewItem.cpp

```diff
@@ -66,7 +66,10 @@ void TabViewItem::OnApplyTemplate()
             if (internalTabView)
             {
                 winrt::ThemeShadow shadow;
-                shadow.Receivers().Append(internalTabView->GetShadowReceiver());
+                if (!SharedHelpers::Is21H1OrHigher())
```

Comment:
If this shadow has no receiver it wont do anything right? can we just not create the shadow at all instead of not setting the receiver?

Reply:
The TabViewItem still needs to have its own ThemeShadow in order for a DropShadow to render. It looks like this block will set `m_shadow`, and  `UpdateShadow()`  then stuffs it in the Shadow property.